### DISCUSSION TITLE
Allow custom directory specification in a project-specific config file for snippets  in Chakra UI CLI

### DISCUSSION THREAD

**Discussion by sangit0** [chakra-ui/chakra-ui#9121] [Ideas]
### Description

Support for saving the directory output dir configuration in a project-specific config file (such as .chakra.config), allowing persistent, reusable directory paths for future snippet additions.

### Problem Statement/Justification

In its current form, the Chakra UI CLI does not allow developers to specify custom directories in a config file which will be across project as directory settings can be shared across the team.

### Support Project-Level Configuration File (.chakra.config):

Introduce a .chakra.config file (or similar) where users can define default settings for the CLI, including the snippet directory. This file could be placed in the root of the project.

`{
  "snippetsOutdir": "src/components"
}`

When adding snippets, the CLI should check for this configuration file and use the specified dir if it exists, allowing users to skip providing the --outdir option each time.

### Additional Information

Benefits:

1. Ease of Use: Once the directory is set in a .chakra.config file, users won’t need to specify the directory each time they add a new snippet, simplifying their workflow.

3. Scalability: This feature allows better scalability for teams, as directory settings can be shared across the team in the .chakra.config file.

**Comment by segunadebayo** [comment]
Pass `--outdir` on `snippet add`. There is no project config file for this yet.

Put the flag in an npm script so the team shares it.
